docs/_toc.yml

```diff
@@ -14,6 +14,41 @@ parts:
       - file: urbanization_comparison
   - caption: Tutorials
     chapters:
-      - file: ../Notebooks/Tutorials/UrbanAreas_tutorials.ipynb
-      - file: ../Notebooks/Tutorials/LEI_Example.ipynb
-
+      - file: notebooks/Tutorials/UrbanAreas_tutorials.ipynb
```

Comment:
@elbeejay Thanks for adding the content. @bpstewar Would you please review?

Reply:
I've commented out everything except the `Tutorials/` notebooks for now, so currently only those two will be included in the built documentation